x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -75,9 +75,32 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
       ))}
       {!isJobCreated && (
         <Fragment>
+          <EuiFormRow
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.jobTypeLabel', {
+              defaultMessage: 'Job type',
+            })}
+            helpText={
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.create.jobTypeHelpText"
+                defaultMessage="This form let's you create an outlier detection job using a model memory limit of 50MByte based on a feature index. To create jobs with more advanced settings or other types of analysis, please switch to the {advancedEditorButton}. Note you cannot switch back to this form once the advanced editor is enabled."
```

Comment:
Typo! Should be `lets`. Also I think `50MB` is good enough - we use e.g. `MB` in other places AFAIK.